src/unix/udp.c

```diff
@@ -533,6 +559,8 @@ int uv__udp_bind(uv_udp_t* handle,
     handle->io_watcher.fd = fd;
   }
 
+  uv__set_recverr(fd);
+
```

Comment:
From the links posted in the issue I'm not sure if this should only be done for `connected` sockets.